symengine/add.cpp

```diff
@@ -290,6 +290,35 @@ RCP<const Basic> add(const RCP<const Basic> &a, const RCP<const Basic> &b)
     return Add::from_dict(coef, std::move(d));
 }
 
+RCP<const Basic> add(const vec_basic &a)
+{
+    if (a.size() < 2) {
+        throw std::runtime_error("add expected at least a vector with size two");
```

Comment:
`add({i2})` is indeed not needed, but say `v` is a vector that you generate and at runtime it contains only one or zero elements. It will be useful in this case.